mix.exs

```diff
@@ -81,7 +83,8 @@ defmodule Phoenix.LiveView.MixProject do
     [
       Introduction: ~r/guides\/introduction\/.?/,
       "Server-side features": ~r/guides\/server\/.?/,
-      "Client-side integration": ~r/guides\/client\/.?/
+      "Client-side integration": ~r/guides\/client\/.?/,
+      "File uploads": ~r/guides\/uploads\/.?/
```

Comment:
I moved the external uploads guide into the `client` dir, but i must not have removed the extra section.

```suggestion
      "Client-side integration": ~r/guides\/client\/.?/
```